pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies=[
 	"pandas",
 	"pooch",
 	"pyarrow",
-	"numpy",
+	"numpy==1.26.0",
```

Comment:
is this really necessary?

Reply:
Yes. torch 2.1.0 is not compatible with numpy 2.0.0.
See https://github.com/pytorch/pytorch/issues/107302

Reply:
Wait, we can also just bump the PyTorch version we use for testing, which sounds more elegant.